src/app.js

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios'
+
+
+//Verwijzingen naar de HTML elementen
+const searchForm = document.getElementById("search-form");
+const infoBox = document.getElementById("country-result");
+
+
+//Event listener naar de submit form
+searchForm.addEventListener('submit', searchCountry)
+
+//Zoekfunctie waar gebruiker land in typt
+function searchCountry(e) {
+    e.preventDefault();
+    const queryfield = document.getElementById("query-field");
+
+    fetchCountryDetails(queryfield.value);
+
+    queryfield.value = ' ';
+
+}
+
+//De Async functie die de data ophaalt
+async function fetchCountryDetails(name) {
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${name}`);
+        const country = result.data[0];
```

Comment:
Je zou hier ook het volgende van kunnen maken
```suggestion
        const { flag, name, population, region, capital, currencies, languages } = result.data[0];
```
Dan hoef je vervolgens nergers meer `country.` vóór te zetten